README.md

```diff
@@ -74,6 +74,24 @@ If you want to change any of these parameters simply add them to a match stanza.
 </match>
 ```
 
+## Kubernetes
```

Comment:
Could you also add a section and a badge from Quay to provide a link to the registry like so:

[![Docker Repository on Quay](https://quay.io/repository/solarwinds/fluentd-kubernetes/status "Docker Repository on Quay")](https://quay.io/repository/solarwinds/fluentd-kubernetes)

Reply:
Yes, will add this.